docs/source/main_classes/logging.mdx

```diff
@@ -54,6 +54,10 @@ verbose to the most verbose), those levels (with their corresponding int values
 - `transformers.logging.INFO` (int value, 20): reports error, warnings and basic information.
 - `transformers.logging.DEBUG` (int value, 10): report all information.
 
+By default, `tqdm` progress bars will be displayed to indicate download progress. 
+`transformers.utils.tqdm_utils.set_progress_bar_enabled` can be used to suppress or unsuppress
+this behavior. 
```

Comment:
```suggestion
By default, `tqdm` progress bars will be displayed to show progress during model download, training in `Trainer` and various other places. [`transformers.utils.tqdm_utils.set_progress_bar_enabled`] can be used to suppress or unsuppress this behavior. 
```

it's not just download progress...

and added auto-link to the doc

Reply:
As it is now, toggling `set_progress_bar_enabled` won't change the behavior of the progress bar in the `Trainer` class. `Trainer` still uses `tqdm.auto.tqdm`.

https://github.com/huggingface/transformers/blob/226e74b610bd8d30c2faba627f245d693ced7b55/src/transformers/trainer.py#L33

For `set_progress_bar_enabled` also take affect in `Trainer`, the above import line should be replaced with `from .utils.tqdm_utils import tqdm`. This is in line with my previous inquiry:

> Should toggling `set_progress_bar_enabled` affect the entire API, for instance the behavior of the Trainer class? This would mean replacing all occurrences of from `tqdm.auto import tqdm` with `from .utils.tqdm_utils import tqdm`.

To my ears, the edit misleadingly suggests otherwise. What do you think? (I should have asked before committing...)

Reply:
I think what you are saying is totally correct, so perhaps add your notes to the logging doc. That's that model download is already controlled by set_progress_bar_enabled, and anywhere else where tqdm.auto.tqdm is used to make it controllable in the same way the user has to swap it for from transformers.utils.tqdm_utils import tqdm